.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: cpp
+cache:
+    directories:
+        - /root/.cache
+sudo: required
+services:
+    - docker
+dist: trusty
+os: linux
+env:
+    - IMAG=apolloauto/apollo:dev-x86_64-20170707_1129
+before_install:
+    - if [ ! -d "/root/.cache" ]; then sudo mkdir /root/.cache; fi
+    - sudo cp scripts/AGREEMENT.txt /root/.cache/.apollo_agreement.txt
+    - docker pull ${IMAG}
+    - docker run -d -it --name apollo_dev -v ${TRAVIS_BUILD_DIR}:/apollo -v /root/.cache:/root/.cache -w /apollo ${IMAG}
+    - docker ps 
+install: skip
+script:
+    - docker exec apollo_dev  bash -c "./apollo.sh clean"
+    - docker exec apollo_dev  bash -c "./apollo.sh build"
```

Comment:
@siyangy Due to our repo does not have esd_can lib, many tests will fail (49 of 74 fails). I only add "build" and "lint" to the CI.